core/validation/failures.py

```diff
@@ -231,7 +214,14 @@ def __str__(self):
         )
 
     def to_user_centered_components(self) -> tuple[str, str, str, str]:
-        return "Unimplemented", "Unimplemented", "Unimplemented", "Unimplemented"
+        sheet, section, column = get_sheet_section_and_column(self)
+
+        message = (
+            f'There are blank cells in column(s): "{column}".'
+            f"Use the space provided to tell us the relevant information."
+        )
+
+        return sheet, section, message
 
```

Comment:
I've seen Elizabeth state that GDS error messages don't end with a full stop

e.g. `Use the space provided to tell us the relevant information.` should be `Use the space provided to tell us the relevant information`

Reply:
Also can column ever be plural? do we need the (s) ?

Reply:
I'll remove the full-stop. 

There's a difficulty with reverse engineering the 'Question' and 'Answer' columns from our dataframe back to what they looked like in the original spreadsheet. If we don't have row_values, all we have is a column called 'Answer' being null, but that corresponds to a series of questions we don't have in the values for. Therefore I've opted to indicate the parts of spreadsheet where the questions are (i.e. A1-A7) in a certain section, hence the (s).